builder/amazon/common/step_security_group.go

```diff
@@ -45,7 +45,9 @@ func (s *StepSecurityGroup) Run(state multistep.StateBag) multistep.StepAction {
 
 	port := s.CommConfig.Port()
 	if port == 0 {
-		panic("port must be set to a non-zero value.")
+		if s.CommConfig.Type != "none" {
+			panic("port must be set to a non-zero value.")
```

Comment:
Thought about this a little overnight.  I'm not a fan of just skipping this and other steps, since every way we make the null communicator less like the others, the less use it likely is for testing.

We can either go with how I have it here, where it just uses port 0 arbitrarily in the call to aws (It's not really doing anything anyway, since we won't be using it; just makes sure the request works as expected), or we can have something in "prepare" that says you have to designate SSHPort or WINRMPort even if you're using the null communicator, just to make the code tick.  

Or we can have something in "prepare" that says we don't support the null communicator with AWS, but maybe that'll take away some debugging options for folks that just keeping an arbitrary port wouldn't.